data/en/test_radio_mandatory_with_mandatory_other.json

```diff
@@ -0,0 +1,59 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A questionnaire to demo the mandatory radio field.",
+    "groups": [{
+        "blocks": [{
+                "type": "Questionnaire",
+                "id": "radio-mandatory",
+                "questions": [{
+                    "type": "General",
+                    "id": "radio-mandatory-question",
+                    "title": "",
```

Comment:
Same as the previous comment, we shouldn't have empty elements, and the question text should be in the question title. This applies to all survey json files in this pull request.